Gaijinizer patch/Scripts.txt

```diff
@@ -2829,90 +2829,85 @@ It is already a limit
- But the 'device' is not that good.
-Although correction can be applied to some extent, it is only that.
- In particular, what you changed was 'the destiny of death' which should have been said to be universal
-It is not a play that can be the guy of that day.
-
-In order to identify that “ability”
-A prototype of the intervention device was distributed as a 'trial version'.
-It will be a long time ago, but do you remember the contents?
-That is 'where' and 'what format',
-Did you 'intervene' in that world .........
-Probably because it was still an experimental device
- I wonder how fragmented it was.
-
-... yeah? Is it time?
-Well then, one last thing!
- It's likely to be misunderstood, but with 'a person who has organized this device'
-'The creator of this game' is probably another person.
- The latter only thinks 'I made a niche eroge'.
-
- 'The guy who put in the equipment' is a que do not know who
-It's probably a resident of 'this world' if it's not a mere entertainer.
- The guys at Hell's Gate have only worked with them.
-It changes her world and tries to let you do something.
-
-But if you say that his journey is over safely ...
-This time, that 'preparing'...may be.
-
-
-Now! Anyway, I'm tired!
-You have successfully completed this game!
- Let's make deep reading a pear, and let's be purely pleased there!
-
-...... If I have something to meet in another place
- At that time, let's get along well again!
- It's a promise!
- =================================================="#MTLed
+ 《Body》==========================================
+...I wonder if this'll work.
+Professor Yui said she'd invented this weird device to be able to
+send messages directly to you, the 'Intervener', so I thought I'd give
+it a try. Apparently the messages come to you in the form of some kind of
+text email.
+
+Well first off, I suppose I should congratulate you. Maybe something
+like, 'Congrats on finishing the game!'. That's right, I know what kind
+of device you've been using to carry out your interventions. That's because
+this isn't the first world that's received very similar intervention.
+There was a world that received similar interventions before. And it's
+always someone from your world who's chosen to intervene.
+
+...Don't worry, I'll keep it a secret from him. ♪
+But I didn't send this message just to congratulate you, I also wanted
+to tell you why you became the 'Intervener'...
+
+Firstly, the fact that you were storing magic while playing in order to
+reach that ending had to be kept under wraps. So that's why your world,
+the world farthest from the Demon Realm was chosen, a world where the
+very notion of 'magic' is nothing but fantasy.
+
+Oh but even though you were storing magic, it was funneled into that
+device you're using, so it won't affect your soul or anything like that.
+Having your soul emptied out just by playing a game would be some kind
+of bad joke.
```

Comment:
Minor flow:
>_**Ah, but even though you were building up magic, it was stored in that
device you're using, so it won't affect your soul or anything like that.
Having your soul sucked out just by playing a game would be some kind
of cruel joke.**_